src/fieri/app/models/collaborator_worker.rb

```diff
@@ -22,11 +22,17 @@ def get_collaborator_count(json)
   def evaluate(cookbook_name)
     cookbook_json = get_json(cookbook_name)
     collaborator_count = get_collaborator_count(cookbook_json)
-    sufficient_collaborators?(collaborator_count)
+    sufficient_collaborators?(collaborator_count) ? false : true
   end
 
   def give_feedback(cookbook_name)
-    evaluate(cookbook_name) ? 'This cookbook has sufficient collaborators.' : 'This cookbook does not have sufficient collaborators.'
+    json = get_json(cookbook_name)
+    collaborator_count = get_collaborator_count(json)
+    if evaluate(cookbook_name)
+      I18n.t('quality_metrics.collaborator.failure', :collaborators => collaborator_count.to_s + ' collaborators', :passing_number => '2')
+    else
+      I18n.t('quality_metrics.collaborator.success', :collaborators => collaborator_count.to_s + ' collaborators')
```

Comment:
Related to other comment: I suggest the interpolated variable here be `num_collaborators` for both messages and move the string "collaborators" to display after the number from here to the I18n strings.

Is `passing_number` something known elsewhere in this class for the evaluation to have occurred?